### ISSUE TITLE
Unwanted month change on keydown pressed

### ISSUE THREAD

**Issue by dialnco** [gpbl/react-day-picker#657]
Hi all! 👋
Think I stumble with a minor bug when using modals and number inputs with react-day-picker.
I think it has to do with keydown events.

### Path to reproduce:
* Use "renderDay" props to render custom day content
* Put a button on day content
* Button opens a Modal with date reference.
* input breaks when using keydown (Up, Down keyboard), changing current Month instead of the input value, closing modal.
* Try the same Modal without date data, and input/calendar doesn't break. Work as expected.

I put a "Trigger Modal" button on top, to open modal without date reference. Works fine.
And there are "click" buttons on each day content, to open modal with date reference. It breaks when keydown (Up, down, e.g).

Made a codesandbox to reproduce:
https://codesandbox.io/s/j24wvkpr03

I have been fighting with this issue for a while, and don't have a clue how to fix it. Any thoughts?
Thanks!
Diego

**Comment by gpbl** [comment]
Mmm I haven’t checked it out yet, but what have you tried so far? I’m thinking for example if stopping some events propagation could help...

**Comment by dialnco** [comment]
I tried whit stopPropagation on events, empty returns, added keydown listeners to catch those events, but nothing so far.
There are the "onDayKeyDown" and "onKeyDown" props on the docs. And haven't got any results with those. 
I think it is more closely related to onDayKeyDown prop.

**Comment by gpbl** [comment]
> I tried with stopPropagation on events

This seems to have fixed it:

```js
<input
   type="number"
   min="0"
   value={this.state.value}
   onKeyDown={e=>e.stopPropagation()}
   onChange={e => this.setState({ value: e.target.value })}
/>
```

https://codesandbox.io/s/219q35mwx0

However, if you don't want to add `stopPropagation` to all the elements in the modal 😄 you could try to put the modal node outside the calendar.

**Comment by dialnco** [comment]
hahaha! I filled all my code with `stopPropagation` but the inputs! 
Can't believe I missed that, thank you!.
You are right! that did the trick. I tried using a few callbacks between components and render the modal outside the calendar as well, and that proved to be a better solution. 

https://codesandbox.io/s/823j14ql8

Anyway, Thanks for the quick response, and Sorry for wasting your time! 😉 
Solved.
